projects/project2/DataGenerationCode/LabProject/Program.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LabProject.DataDedup.DataGenerator
+{
+    class Program
```

Comment:
Not sure if this C# code is needed for the project? In case you have used it for experiments only, please, consider removing it.

Reply:
We couldn't get the Data, so we generated it ourself. The C# code was for Data Generation. We will delete it.

Reply:
@haseebjadoon commited the code, he is on his way to Pakistan now. He will update it once he reaches Pakistan.

Reply:
can we update the report after the presentation or do have to do it before?

Reply:
Please, do it before. We will not evaluate it if you submit after the presentation. We will consider only the merged PR for our evaluation.

Reply:
Noted. We will fix the format of PR too.